.travis.yml

```diff
@@ -3,11 +3,21 @@ language: go
 go:
   - 1.11.x
   - 1.12.x
+  - 1.13.x
+  - 1.14.x
```

Comment:
can be a separate PR

Reply:
I guess I can do that, but there are some changes in the way go modules does vendoring, making tests fail for v1.13 and v1.14 if running `go mod vendor` in v1.12, and tests failing in v1.14 if running `go mod vendor` with v1.13.

https://golang.org/doc/go1.14#go-command

Do you still think I should split it?

Reply:
Please split. We may need some additional work for Go 1.14, because it changed the scheduler and EINTR stuff: https://golang.org/doc/go1.14#runtime

Reply:
Ahh, thant makes sense, thanks! Going back to v1.13 then.

Reply:
We faced the very same issue and provide a `vendor-in-container` target to force _one_ version of go for vendoring (see https://github.com/containers/libpod/blob/master/Makefile#L633).